cumulusci/cli/config.py

```diff
@@ -19,15 +19,20 @@ def __init__(self, *args, **kwargs):
         try:
             super(CliRuntime, self).__init__(*args, **kwargs)
         except (ProjectConfigNotFound, NotInProject) as e:
-            raise click.UsageError(str(e))
+            raise click.UsageError("{}: {}".format(e.__class__.__name__, e))
```

Comment:
What's the purpose of adding the class name here? It feels wonky to put it in front of users in cases where we're handling an expected exception that has a decently understandable message.

Reply:
That's a leftover from a prior implementation I was working on.  I'll reset it.